### PR TITLE
Week1

### PR DESCRIPTION
**Pull request by Shabnam14**
Completed assignment for week 1

### CODE REVIEW

File: Week-1/Assignment1.2/index.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    </div>
+    <img id="ifiuh" src="https://uploads.codesandbox.io/uploads/user/a8233562-b36b-483d-9de9-5a1ac148fb0c/2Eq--cattyboard.jpg"/>
+    <div style="position: absolute; padding-left: 280px; margin-top: -25px;">
+    	<i class="fa-solid fa-play" style="color:white;"><t style="color:white; font-size: 12px; padding-left: 8px;">Play all</t></i>
+    	<i class="fa-solid fa-plus" style="color:white; padding-left: 20px;"><t style="color:white; font-size: 12px; padding-left: 8px;">Add to</t></i>
+    	<i class="fa-solid fa-ellipsis" style="color:white; padding-left: 20px;"><t style="color:white; font-size: 12px; padding-left: 8px;">More</t></i>
+    </div>
+  </header>
+  	<body>
+		<body id="ibw9">
```
Comment (dev-ashishk) [inline Week-1/Assignment1.2/index.html]: why have you used this kind of `id` and `class` names ? any specific reason ?
follow `BEM` convention. http://getbem.com/

File: Week-1/Assignment1.2/index.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+          					<td class="tg-yjjc"><a href="https://google.com/" target="_blank" rel="noopener noreferrer">CATTY CLOUD SYNC</a></td>
+        				</tr>
+      				</tbody>
+    			</table>
+  			</div>
+  				<div class="row" id="igqj1">
+  					<div style="padding-left: 600px; padding-top: 40px; position: absolute;">
+  					<i class="fa-solid fa-backward-step" style="color:white;"></i>
+  					<i class="fa-solid fa-pause" style="padding-left: 100px; color: white;"></i>
+  					<i class="fa-solid fa-forward-step" style="padding-left: 100px; color: white;"></i>
```
Comment (dev-ashishk) [inline Week-1/Assignment1.2/index.html:58]: let's try not to write inline css, when we already have an external css file. 
Create an appropriate css class and use it here.

File: Week-1/Assignment1.2/index.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  					<i class="fa-solid fa-rotate-right" style="padding-left: 100px; color: white;"></i>
+  					<i class="fa-solid fa-shuffle" style="padding-left: 100px; color: white;"></i>
+  					</div>
+    				<div class="cell">
+    					<img id="i8ydg" src="https://uploads.codesandbox.io/uploads/user/a8233562-b36b-483d-9de9-5a1ac148fb0c/2Eq--cattyboard.jpg"/>
+    					<div style="position: absolute; margin-top: -100px; padding-left: 110px;"><t style="color:white; font-size: 12px; padding-left: 8px;">If It Ain't Love</t>
```
Comment (dev-ashishk) [inline Week-1/Assignment1.2/index.html]: what does this tag `<t> </t>` do ?

**Review by dev-ashishk — CHANGES_REQUESTED** [review]

File: Week-1/Assignment1.2/index.html (hunk 3 above)
Comment (Shabnam14) [reply Week-1/Assignment1.2/index.html]: only to enhance the text, removed as it didnt have much importance

File: Week-1/Assignment1.2/index.html (hunk 2 above)
Comment (Shabnam14) [reply Week-1/Assignment1.2/index.html:58]: Created a new class

File: Week-1/Assignment1.2/index.html (hunk 1 above)
Comment (Shabnam14) [reply Week-1/Assignment1.2/index.html]: Changed all classes and ids

**Review by dev-ashishk — APPROVED** [review]

**Pull request merged**